sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -64,6 +67,7 @@ def generate_blob(
         content_encoding: Optional[str] = None,
         content_language: Optional[str] = None,
         content_type: Optional[str] = None,
+        sts_hook: Optional[Callable[[str], Any]] = None,
```

Comment:
I think the type should be `Optional[Callable[[str], None]]` (`None` return instead of `Any`)? Did Anna suggest this type?

Reply:
`Optional[Callable[[str], None]]` makes more sense since we're not utilizing the return value from the hook. Will take this feedback.